test/Microsoft.AspNetCore.Sockets.Client.Tests/LongPollingTransportTests.cs

```diff
@@ -201,5 +207,77 @@ public class LongPollingTransportTests
                 }
             }
         }
+
+        [Fact]
+        public async Task LongPollingTransportDispatchesMessagesReceivedFromPoll()
+        {
+            var message1Payload = new byte[] { (byte)'H', (byte)'e', (byte)'l', (byte)'l', (byte)'o' };
+            var message2Payload = new byte[] { (byte)'W', (byte)'o', (byte)'r', (byte)'l', (byte)'d' };
+            var encoded = Enumerable.SelectMany(new[] {
+                new byte[] { (byte)'B', 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x05, 0x00 }, message1Payload,
```

Comment:
Yep, I didn't update them when I removed the Fin flag, which was on the high bit. Updating in https://github.com/aspnet/SignalR/pull/305